app/pages/lab/helpers/subject-sets/cleanSubjectData.js

```diff
@@ -0,0 +1,16 @@
+import constants from './constants'
+
+const { INDEX_FIELD_HEADER } = constants;
+
+export default function cleanSubjectData(rawData) {
+  const cleanData = {};
+  for (const [key, value] of Object.entries(rawData)) {
+    let cleanKey = key.trim();
+    let cleanValue = value.trim ? value.trim() : value;
+    if (cleanKey.startsWith(INDEX_FIELD_HEADER)) {
+      cleanKey = cleanKey.slice(1);
```

Comment:
[minor] A small suggestion - this can be changed to `cleanKey = cleanKey.slice(INDEX_FIELD_HEADER.length)`

Reply:
Good idea.